app/assets/javascripts/tooltips.coffee

```diff
@@ -1,12 +1,7 @@
 # Pertaining to tooltips
 $(document).on 'turbolinks:load', ->
   # For regular spans
-  $('.daria-tooltip').each ->
-    $(@).tooltip({
-      html: true,
-      placement: 'bottom',
-      title: $(@).data('tooltip-text')
-    })
+  $('.daria-tooltip').tooltip()
```

Comment:
setting this via data attributes is a little closer to the metal, so we do that there.